antismash/common/hmm_rule_parser/cluster_prediction.py

```diff
@@ -762,8 +762,6 @@ def detect_protoclusters_and_signatures(record: Record, ruleset: Ruleset,
             annotate_existing_subregions: if True, subregions already present in the record
                     will have domains annotated even if no protocluster is found
             multipliers: distance multipliers to apply to rules
-            rule_subset: an optional subset of rule identifiers within the rules to
```

Comment:
Looking at this, `multipliers` should also have been removed, so I've fixed that.
I've shuffled some other parts around as well, since there were a couple of test failures and some typing errors in the interim commits.